main.js

```diff
@@ -53,16 +55,20 @@ function playRound(playerSelection, computerSelection) {
     return "Oh noes!";
   }
 }
-function tally() {
-  if (playRound().includes(subStringWin)) {
+function tally(roundResult) {
+  if (roundResult.includes(subStringWin)) {
```

Comment:
because id now need to define roundresult and use the return of each if statement to be stored in roundresult?